src/psycop_feature_generation/loaders/raw/load_text.py

```diff
@@ -240,3 +235,37 @@ def load_arbitrary_notes(
         text_sfi_names,
         n_rows=n_rows,
     )
+
+
+@data_loaders.register("preprocessed_sfis")
+def load_preprocessed_sfis(
+    text_sfi_names: str | list[str] | set[str] | None = None,
```

Comment:
```suggestion
    text_sfi_names: str | list[str] | set[str] | None = None,
```

This type hint is a bit messy; do we need to accept both str, list and set?

Two proposed alternatives:

* `Sequence[str] | None` <- would prefer, if only one name then people should just specify e.g. `[sfi_name_one]`. Makes the internals of `load_preprocessed_sfis` easier to reason about. Alternatively:
* `Sequence[str] | str | None`